nautobot/core/jobs/bulk_actions.py

```diff
@@ -46,11 +48,25 @@ def _update_objects(self, model, form, filter_query_params, edit_all, nullified_
             filterset_cls = get_filterset_for_model(model)
 
             if filter_query_params and not filterset_cls:
-                self.logger.error(f"Filterset not found for {model}")
-                raise RunJobTaskFailed(f"Filter query provided but {model} do not have a filterset.")
+                self.logger.error(f"Filterset not found for `{model}`")
+                raise RunJobTaskFailed(f"Filter query provided but `{model}` do not have a filterset.")
+
+            # Discarding non-filter query params
+            new_filter_query_params = {}
```

Comment:
Are/can we discard the non-filter params before they even get to this job? Hoping this logic here is just extra protection, not the primary line of defense.

Reply:
Currently implemented it is the primary line of defense. I will see what I can do in the views.